docs_espressif/en/additionalfeatures/heap-tracing.rst

```diff
@@ -0,0 +1,36 @@
+Heap Tracing
+========================
+
+Heap Tracing allows tracing of code which allocates/frees memory. More information in `Heap tracing documentation <https://docs.espressif.com/projects/esp-idf/en/latest/api-reference/system/heap_debug.html#heap-tracing>`_. Please also review `System behaviour analysis <https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/app_trace.html#system-behavior-analysis-with-segger-systemview>`_ for systemView tracing configuration.
+
+Let's open a ESP-IDF project. For this tutorial we will use the ``system/sysview_tracing_heap_log`` example.
+
+Click menu **View** -> **Command Palette...** and search for the **ESP-IDF: Show Examples Projects** command and choose ``Use Current ESP-IDF (/path/to/esp-idf)``. If the user doesn't see the option, please review the setup in :ref:`Install ESP-IDF and Tools <installation>`.
```

Comment:
For consistency in addressing the reader, I would suggest addressing him directly and not use the `user` word here